cli/createKeychainCmd.go

```diff
@@ -38,7 +38,11 @@ func GetCreateKeychainCmd() *cobra.Command {
 	createKeychainCmd.Flags().String("access-seed", "", "Access Seed")
 	createKeychainCmd.Flags().Bool("ssh", false, "Enable SSH key mode")
 	createKeychainCmd.Flags().String("ssh-path", GetFirstSshKeyDefaultPath(), "Path to ssh key")
+	createKeychainCmd.Flags().Bool("bip39", false, "Enable BIP39 words for seed")
```

Comment:
why do you need to specify a flag with bip39 can't you interpret it like in the TUI ?
in anycase, the ssh key cannot be bip39

Reply:
the goal of the flag is to display a prompt for the user to type the words 🤔
(so the words list is not displayed in the terminal history) 
but if you prefer I can also try to interpret the value of `access-seed` flag?

Reply:
Maybe the both behavior could be interesting